src/goto-symex/symex_target_equation.h

```diff
@@ -405,6 +405,9 @@ class symex_target_equationt:public symex_targett
   // for enforcing sharing in the expressions stored
   merge_irept merge_irep;
   void merge_ireps(SSA_stept &SSA_step);
+
+  // for unique I/O names
+  std::size_t io_count = 0;
```

Comment:
Ok, but possibly `symex_target_equationt` really never was the right place to do this? To me, this looks just like what we're doing with renaming nondets, and I am thus wondering whether `goto_symext` would really be the right place to take care of this?